### ISSUE TITLE
Explorer Patcher Windows Apps Disappear

### ISSUE THREAD

**Issue by CMMD77** [valinet/ExplorerPatcher#1625]
All Windows apps (Basically from Store) disappear when Start menu is set to Windows 10.  No other option selected.  If change back to Windows 11 apps show up.

The Windows store also has issues that it gets errors that Edge browser needs to be updated even though it is as up to date as it gets.  This appears where install button should be.  Only fix for this so far has been to uninstall Explorer Patcher, clean registry and files manually as uninstall leaves a lot behind that causes issues.  Run repair in safe mode on windows or skip all that and reset windows.  EG:  Plex from store gets this error in store only after Explorer Patcher install.

Any help would be appreciated.

**Comment by CMMD77** [comment]
FYI.  This does not happen on older versions.  But if you install latest version you cannot go back to previous version as it leaves problem behind.

**Comment by Vipin2003** [comment]
Check #1591 for fix and close the issue.

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1108

**Comment by Poopooracoocoo** [comment]
Closing as duplicate of #1108
